imports/client/components/SetupPage.tsx

```diff
@@ -459,7 +458,7 @@ const FeatureToggle = ({ enabled, onToggleEnabled }: {
 
 const GoogleIntegrationSection = () => {
   const enabled = useTracker(() => !Flags.active('disable.google'), []);
-  const oauthSettings = useTracker(() => ServiceConfiguration.configurations.findOne({ service: 'google' }) as any, []);
+  const oauthSettings = useTracker(() => ServiceConfiguration.configurations.findOne({ service: 'google' }) as unknown as { clientId: string }, []);
```

Comment:
Should this be `as unknown as undefined | { clientId: string }`?  There's no guarantee we've got a record in the DB to retrieve for `service: 'google'`, so we shouldn't assert that it's always going to return an object.